test/golden/layout_demo.txt

```diff
@@ -0,0 +1,30 @@
+[-1;-1H[0m    [0;38;2;143;188;187;48;2;46;52;64m                                             [0m                                                   [0m
```

Comment:
I like using golden files 👍 

Let's see if this approach scales. (If it turns out that it's too brittle we might need to build a little simulator/interpreter that reads the stdout bytes and interprets the control characters and the simulates what the terminal would do. Then we could have a "print-screen" of the terminal after the control characters have been interpreted. But maybe this approach is already good enough.)